### PR TITLE
Written README file

### PR DESCRIPTION
**Pull request by MohammedAbuSamra**
relates #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+
+
+## What is Finance App?
+
+
+## What is the technologies I used to create this website?
+
+- HTML5 & CSS3
+- Responsive Web Design
+- Git & Github
```
Comment (moustf) [inline README.md:18]: The technologies should be in the 'How?' section that tells the reader how we did create this app.

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
\ No newline at end of file
+# finance-app-team2
+## You can check my portfolio [HERE]()
+
+---
+## Team Members:-
+- [Mohammed Ghassan](https://github.com/MohammedAbuSamra)
+
+## Why I created this website?
+
```
Comment (moustf) [inline README.md:9]: Why did **we** create this app?

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
-# finance-app-team2
\ No newline at end of file
+# finance-app-team2
+## You can check my portfolio [HERE]()
```
Comment (moustf) [inline README.md:2]: You can check out the app from [here]()
Edit portfolio

**Review by moustf — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MohammedAbuSamra) [reply README.md:9]: Ok, it's for  you

**Review by moustf — CHANGES_REQUESTED** [review]
The changes in my message.
Thanks.

**Review by moustf — APPROVED** [review]
Good job Abo Gassan👏

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by ahlam377 — APPROVED** [review]

**Pull request merged**